ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/precompiles/privacy/PrivacyPrecompiledContract.java

```diff
@@ -65,6 +66,10 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(PrivacyPrecompiledContract.class);
 
+  static final PrecompileContractResult NO_RESULT =
```

Comment:
What if we defined this as a static on PrecompileContractResult? That would minimize duplication and further divorce from privacy stuff.

Reply:
This is unique to the privacy precompiles, as they set refund gas to true.